charts/values.yaml

```diff
@@ -47,6 +47,14 @@ mayastor:
 
 # -- Configuration options for pre-upgrade helm hook job.
 preUpgradeHook:
+  # -- Node tolerations for server scheduling to nodes with taints
+  ## Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/
+  ##
+  tolerations: []
+  # -- Optional array of imagePullSecrets containing private registry credentials
+  ## Ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+  imagePullSecrets: []
```

Comment:
Looks like we have an image key entry below, so we could put it there?
ex: `image.pullSecrets`

Reply:
Yeah, we could. But I think the best practice is to put `imagePullSecrets` at the pod level. I found this example:
[https://github.com/cetic/helm-job/blob/master/templates/job.yaml](https://github.com/cetic/helm-job/blob/master/templates/job.yaml)
But if you prefer, I can put it under the `image` section instead.

Reply:
Yes that makes sense, let's leave it as is then